plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/BaseRaptorConnectorTest.java

```diff
@@ -294,12 +383,12 @@ public void testShardsSystemTable()
                         "SELECT table_schema, table_name, sum(row_count)\n" +
                         "FROM system.shards\n" +
                         "WHERE table_schema = 'tpch'\n" +
-                        "  AND table_name IN ('orders', 'lineitem')\n" +
+                        "  AND table_name IN ('orders', 'region')\n" +
```

Comment:
to improve test runtime?

Reply:
To use a table in `REQUIRED_TPCH_TABLES`. `lineitem` table doesn't exist.
https://github.com/trinodb/trino/blob/master/testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java#L53